pandas/tests/groupby/test_function.py

```diff
@@ -585,6 +585,34 @@ def test_ops_general(op, targop):
     tm.assert_frame_equal(result, expected)
 
 
+def test_ops_not_as_index(reduction_func):
+    # GH 10355
+    # Using as_index=False should not modify grouped column
+
+    # GH 5755
+    if reduction_func in (
+        "corrwith",
+        "idxmax",
+        "idxmin",
+        "mad",
+        "ngroup",
+        "nth",
+        "nunique",
+        "size",
+        "skew",
+    ):
+        pytest.skip("Skip until #5755 is resolved")
```

Comment:
can you give some more text on what is the issue in 5755

Reply:
I broke out the tests which I think have a clear desired behavior but fail from the ones where 5755 needs a resolution, and added a description for each.